Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -209,26 +212,38 @@ public enum SentrySDK {
         let exceptionInfo = exceptionRecord.ExceptionInformation
         // For stowed exceptions, the first element in `ExceptionInformation` is a pointer to an array of `STOWED_EXCEPTION_INFORMATION_V2`
         // and the second element is the total number of stowed exceptions in this array
+        var hresult: String?
         if let arrayPointer = UnsafeMutablePointer<UnsafeMutablePointer<STOWED_EXCEPTION_INFORMATION_V2>?>(bitPattern: UInt(exceptionInfo.0)) {
             let totalExceptions = Int(exceptionInfo.1)
              // Loop from end to beginning to put the last stowed exception at the end of the list, as it's the most recent
              // one and that's what Sentry will display first.
             for index in (0..<totalExceptions).reversed() {
```

Comment:
Can you also limit the number of exceptions reported as part of this improvement?

Reply:
I'd prefer doing this separately as it's orthogonal to what I'm doing here.

Reply:
sounds good - one could argue that the modulecache is also orthogonal :-), but a separate PR lgtm.

Reply:
Haha yeah, this PR is about fixing the broken stuff. The follow up will add new features!